templates/homepage-tab-menu-band.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="grid-wrapper homepage-tab-menu-band">
+  <div class="width-12-12 width-12-12-m">
+    <div class="tab-menu-row">
+      <div class="tab-menu-box">
+        <!-- Automate menu button -->
+        <a href="#{{ homepage.automate.section_anchor }}" target="_blank">
```

Comment:
I don't think these should have target-blank, it's opening the links in a new tab instead of scrolling down.

Reply:
ah... great catch. I think this was a copy/pasta mistake. thanks!